src/test/scss/scss.spec.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * This is a test setup file that enables Jest to run Scss unit tests using the sass-true library
```

Comment:
Nit: As this is a setup file, would it make more sense to name it `scss.setup.js`, or is this the only way to get it to run?

Reply:
I might be able to get that to work but it would require tweaking Jest to look for a file matching that pattern. Personally I'd prefer to keep the `.spec` but what if I renamed it to something like `scss-setup.spec.js`?